data/json/items/books.json

```diff
@@ -3125,6 +3125,50 @@
     },
     {
         "type" : "BOOK",
+        "id" : "necropolis_freq",
+        "name" : "frequency list",
+        "name_plural" : "Frequency Lists",
```

Comment:
Not sure why the singular form is "frequency list" but the plural form is "Frequency Lists". I assume it's an oversight?